### PR TITLE
Cfy 4806 secured communication over internal port

### PR DESCRIPTION
**Pull request by noakup**

### CODE REVIEW

File: cloudify_cli/bootstrap/bootstrap.py [hunk 1]
```diff
@@ -222,11 +218,9 @@ def get_protocol(rest_port):
             manager_node_instance=manager_node_instance)
 
         _upload_resources(manager_node, fabric_env, manager_ip, rest_port,
```
Comment (noakup) [inline cloudify_cli/bootstrap/bootstrap.py]: should use external protocol

File: cloudify_cli/bootstrap/bootstrap.py [hunk 2]
```diff
@@ -234,7 +228,7 @@ def get_protocol(rest_port):
         'manager_user': manager_user,
         'manager_key_path': manager_key_path,
         'rest_port': rest_port,
-        'protocol': protocol
+        'protocol': rest_protocol
     }
```
Comment (noakup) [inline cloudify_cli/bootstrap/bootstrap.py]: use external protocol

File: cloudify_cli/bootstrap/bootstrap.py [hunk 3]
```diff
@@ -396,6 +394,17 @@ def _upload_provider_context(remote_agents_private_key_path, fabric_env,
         fabric.run(upload_provider_context_cmd)
 
 
+def _get_manager_protocol(manager_node):
```
Comment (noakup) [inline cloudify_cli/bootstrap/bootstrap.py]: add todo comment

**Comment by geokala** [comment]
Closed: As this PR is more than six months old without having been merged. Please reopen (and rebase if necessary) if it is still relevant.

**Pull request closed without merging**